uint64 timestamp			# microseconds since system boot, needed to integrate
float32 indicated_airspeed_m_s		# indicated airspeed in meters per second, -1 if unknown
float32 true_airspeed_m_s		# true airspeed in meters per second, -1 if unknown
float32 air_temperature_celsius		# air temperature in degrees celsius, -1000 if unknown
